CHANGELOG.md

```diff
@@ -33,6 +33,12 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Added support for AWS provider.
 
+## [v0.4.1]
+
+### Added
+
+- Add AWS support (#5)
```

Comment:
Does this "#5" have a meaning? If yes, I would add a link for that. Else we can just delete it, I guess.

Reply:
It was supposed to be a link to PR. Will remove it from here, it's somewhat already included in the release. Release changelog link leads to release.
```suggestion
- Add AWS support
```

Reply:
@webwurst PTAL